<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGraphAppClass</class>
 <widget class="QMainWindow" name="QtGraphAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGraphApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GraphView" name="graphScreen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1021</width>
      <height>561</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yPos">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNewNode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>610</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(90, 90, 90);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qtgraphapp.qrc">
      <normaloff>:/QtGraphApp/Resources/NewNodeIconTransparent.png</normaloff>:/QtGraphApp/Resources/NewNodeIconTransparent.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConnection">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>610</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(90, 90, 90);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qtgraphapp.qrc">
      <normaloff>:/QtGraphApp/Resources/NewConnectionIcon.png</normaloff>:/QtGraphApp/Resources/NewConnectionIcon.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHi">
    <property name="title">
     <string>Hi</string>
    </property>
   </widget>
   <addaction name="menuHi"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header>graphview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtgraphapp.qrc"/>
 </resources>
 <connections/>
</ui>
